src/genie/libs/parser/iosxr/show_mrib.py

```diff
@@ -352,3 +352,106 @@ def cli(self, vrf='', af='',output=None):
                 continue
 
         return parsed_dict
+
+
+# ==========================================================================
+# Schema for 'show mrib evpn bucket-db'
+# ==========================================================================
+class ShowMribEvpnBucketDbSchema(MetaParser):
+    """ Schema for show mrib evpn bucket-db. """
+
+    schema = {
+        'bucket_id':
+            {int:
+                {'if_handle': str,
+                 'if_name': str,
+                 'delete_in_progress': str,
+                 'state': str,
+                 'uptime': str
+                 },
+             },
+    }
+
+
+# ==========================================================================
+# Parser for 'show mrib evpn bucket-db'
+# ==========================================================================
+class ShowMribEvpnBucketDb(ShowMribEvpnBucketDbSchema):
+    """
+    Parser for show mrib evpn bucket-db.
+
+    Parameters
+    ----------
+    device : Router
+        Device to be parsed.
+
+    Returns
+    -------
+    parsed_dict : dict
+        Contains the CLI output parsed into a dictionary.
+
+    Examples
+    --------
+    >>> show_mrib_evpn_bucket_db(uut1)
+
+    {'bucket_id':
+        {0:
+            {'if_handle': '0x2007ae0',
+             'if_name': 'Bundle-Ether1',
+             'delete_in_progress': 'N',
+             'state': 'Forward',
+             'uptime': '02:24:24'
+            },
+        1:
+            {'if_handle': '0x2007ae0',
+             'if_name': 'Bundle-Ether1',
+             'delete_in_progress': 'N',
+             'state': 'Blocked',
+             'uptime': '02:24:24'
+            },
+        2: ...
+        }
+    }
+
+    """
+
+    cli_command = ["show mrib evpn bucket-db"]
+
+    def cli(self, output=None):
+
+        if output is None:
+            out = self.device.execute(self.cli_command[0])
+        else:
+            out = output
+
+        parsed_dict = {}
+
+        # IFName   IFHandle   BucketID    State    Uptime    Delete In Progress
+        p1 = re.compile(r"IFName +IFHandle +BucketID +State +Uptime +Delete +In +Progress")
+
+        # Bundle-Ether1   0x400b920   0   Forward   01:56:04   N
+        p2 = re.compile(r"(?P<if_name>\S+) +(?P<if_handle>0x[a-fA-F\d]+) +"
```

Comment:
`(?P<if_handle>0x[a-fA-F\d]+)` can be simplify like this  `(?P<if_handle>[\w]+)`

Reply:
Seems you miss this comment, any update on this one?

Reply:
@LenzDong Sorry missed this one, reason I kept as [a-fA-F\d]+ is because it should only be a hex value so only a-f and digits, not all possible values on \w. Could change to \w though, let me know what you think, thanks!

Reply:
ok, I see. We can keep it as this format. Thanks